README.md

```diff
@@ -0,0 +1,23 @@
+# Urban Airship Accengage Cordova Plugin
+
+This plugin supports Cordova apps running on both iOS and Android.
+
+### Issues
+
+Please visit http://support.urbanairship.com/ for any issues integrating or using this plugin.
+
+
+### Requirements:
+ - cordova >= 9.0.0
+ - cordova-ios >= 5.0.1
+ - cococapods >= 1.7.3
```

Comment:
It also requires the urbanairship-cordova package, right?

Reply:
yes, you are right